Update README.grid_fdda - add (max_dom) for rinblw

pengzimu opened the pull request:
TYPE: text only

KEYWORDS: readme, rinblw, max_dom

SOURCE: Peng Zimu (Peking University) and internal

DESCRIPTION OF CHANGES: 
One of the FDDA capabilities for blending observations with the gridded simulation relies on the appropriate 
observation density within a specified radius of influence, which is controlled by the option rinblw (km). To allow 
for different observations on different nests, this option is defined in the Registry as a max_domains namelist entry. 
Therefore this option should be specified for each individual domain. This PR:

1. adds (max_dom) to rinblw in test/em_real/README.grid_fdda
2. adds multiple columns for rinblw in test/em_real/examples.namelist. 

For users with nested domains, before this mod, the WRF code returned:
```
Error in rinblw, please specify a reasonable value ***
```
This update to the documentation helps users to _a priori_ correctly set rinblw for nested domains. 

LIST OF MODIFIED FILES:
M   test/em_real/examples.namelist
M   test/em_real/README.grid_fdda

TESTS CONDUCTED: 
1. After setting rinblw for nested domains, the WRF with surface-analysis nudging can run successfully.
2. Only README files changed, no tests required.

RELEASE NOTES: A modification to a couple of the README files, which helps users to correctly set rinblw when surface-analysis nudging is applied to nested domains. Previously, the rinblw was not consistently identified as a max_dom variable.

smileMchen commented:
This modification is correct and necessary. If no injection, I would like to approve this PR.

davegill commented:
@pengzimu 
1. Please fill in the rest of the PR commit message.
2. Also, the test/em_real/examples.namelist should have rinblw in multiple columns
3. Describe the before and after status of your run

davegill commented:
@smileMchen @pengzimu 
Ming,
Let's get the PR message fixed, and the test/em_real/examples.namelist modified before the commit.

smileMchen commented:
@davegill @pengzimu 
I have changed examples.namelist and modified the PR message.

davegill commented:
@pengzimu @smileMchen 
We would like your affiliation please, so that we may reference you in our release notes, and so that your name is on this commit message.

pengzimu commented:
@davegill @smileMchen 
I have added my affiliation and test information to the PR commit message. And I also changed the word "observational nudging" to "surface-analysis nudging" in the release notes since the option rinblw belongs to surface-analysis nudging according to the test/em_real/examples.namelist.

davegill commented:
@pengzimu 
Peng,
Did you get an email from jenkins? Maybe in your spam folder? If so, please include the text of that email in one of these comment boxes below the commit message.

pengzimu commented:
@davegill 
Up to now I haven't found any email from jenkins in my inbox or spam folder.
